server/config/ckb_node_hosts.yml

```diff
@@ -0,0 +1,6 @@
+hosts:
+  - "http://47.111.175.189:8121"
```

Comment:
We should not show these node urls to public.

Reply:
Remove this file from git

two problems:
1. those nodes are not public, you expose them
2. not easy to run another explorer for another chain